aquifers/AQUFLUX-02.DATA

```diff
@@ -0,0 +1,307 @@
+--
+-- PROPS SECTION
+--
+-- =================================================================================================================================
+PROPS
+--
+--    Input of fluid properties and relative permeability
+--
+SWOF
+--     	Sw	        Krw	        Kro	     Pc
+	0.000000	0.000000	1.00000	    3.75633
+	0.0500000	0.000860000	0.847820	1.86981
+	0.100000	0.00263000	0.697460	1.23731
+	0.150000	0.00524000	0.557170	0.91821
+	0.200000	0.00877000	0.432860	0.72451
+	0.250000	0.0133800	0.327570	0.59341
+	0.300000	0.0192700	0.241770	0.49811
+	0.350000	0.0267200	0.174150	0.42511
+	0.400000	0.0360800	0.122370	0.36691
+	0.450000	0.0478100	0.0837400	0.31911
+	0.500000	0.0625000	0.0556500	0.27881
+	0.550000	0.0809000	0.0357200	0.24401
+	0.600000	0.103940	0.0219900	0.21351
+	0.650000	0.132770	0.0128400	0.18631
+	0.700000	0.168690	0.00699000	0.16161
+	0.750000	0.213020	0.00346000	0.13901
+	0.800000	0.266670	0.00149000	0.11801
+	0.850000	0.329180	0.000510000	0.09831
+	0.900000	0.397060	0.000120000	0.07961
+	0.950000	0.461030	0.00001	0.06161
+	1.00000	0.500000	0.000000	0.04408
+/
+
+PVTW
+    277.0      1.038      4.67E-5    0.318       0.0 /
+
+PVCDO
+    277.0        1.001      0.0          835.0    0.0
+/
+
+ROCK
+      277.0     4.84E-5 /
+
+DENSITY
+      860.04 1033.0    0.853  /  Justert 22/7
+
+NOECHO
+-- =================================================================================================================================
+--
+-- SOLUTION SECTION
+--
+-- =================================================================================================================================
+SOLUTION
+
+EQUIL
+-- Datum    P     woc     Pc   goc    Pc  Rsvd  Rvvd
+   2525.   270   2600    0.0  2200.  0.0   1*   0   0/
+
+
+AQUANCON
+-- Aq#     I1 I2  J1   J2  K1 K2 FACE
+  1     1   2    1   1   1   5  I-    0.8         1*         NO/
+  2     19  20   1   1   1   5  I+    1.         1*         NO/
+/
+
+
+
+DEBUG
+12 /
+
+
+RPTRST
+BASIC=2 KRO KRW KRG PRES /
+
+RPTSOL
+FIP=3  SWAT /
+
+-- =================================================================================================================================
+--
+-- SUMMARY SECTION
+--
+-- =================================================================================================================================
+SUMMARY
+--
+--       EXPORT STANDARD SUMMARY VARIABLE VECTORS TO FILE
+--
+ALL
+--
+--       ACTIVATE EXCEL SUMMARY OUTPUT FILE OPTION
+--
+EXCEL
+
+-- ---------------------------------------------------------------------------------------------------------------------------------
+-- ANALYTICAL AQUIFER INFLUX VARIABLES
+-- ---------------------------------------------------------------------------------------------------------------------------------
+--
+--       ANALYTICAL AQUIFERS(FIELD)
+--
+FAQR
+FAQT
+--
+--       ANALYTICAL AQUIFERS
```

Comment:
We don't really support field-level aggregation of aquifer quantities yet so this test case won't write any non-trivial vectors to the summary file.  Maybe we could request `AAQR` and `AAQT` in addition to the field-level quantities?

Reply:
Sure. I will do that.

Reply:
I think AAQR and AAQT are covered by `ALL` , maybe we should remove FAQR and FAQT, they are also covered by `ALL`.

Reply:
> I think AAQR and AAQT are covered by `ALL`

Oh, yeah, you're right.  I didn't see `ALL` there.  That will absolutely cover the aquifer-related vectors so we don't have request those specifically.